index.html

```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
+                    <p class="disc">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="tonicl">
+                        <ul class="bottom-list">
+                            <li><button class="language-buttons" type="button">html</button></li>
+                            <li><button class="language-buttons" type="button">css</button></li>
+                            <li><button class="language-buttons" type="button">javaScript</button></li>
+                        </ul>
+                    
+                              <div class="btn">
+                                <button type="submit" class="see">See projects</button>
+                              </div>
+                            </ul>
+                </div>            
+                    </section>
+                    <section class="work">
+                        <div class="image">
+                            <img src="./images/snap4.png" alt="snap4"/>
+                        </div>
+                        <div class="content">
+                            <h2>Tonic</h2>
+                            <div class="event2-tonic">
+                                <div class="word">
+                                  <h3>CANOPY</h3>
+                                </div>
+                                <img src="./images/Counter.png" alt="counter image" />
+                                <div class="role">
+                                  <h3>Back End Dev</h3>
+                                </div>
+                                <img src="./images/Counter.png" alt="counter image" class="dot" />
+                                <div class="year">
+                                  <h3>2015</h3>
+                                </div>
+                              </div>
+                            </div>
+                            <p class="disc">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+                            <ul class="tonicl">
+                                <ul class="bottom-list">
+                                    <li><button class="language-buttons" type="button">html</button></li>
+                                    <li><button class="language-buttons" type="button">css</button></li>
+                                    <li><button class="language-buttons" type="button">javaScript</button></li>
+                                </ul>
+                            
+                                      <div class="btn">
+                                        <button type="submit" class="see">See projects</button>
+                                      </div>
+                                    </ul>
+                        </div>
+                        
+                            </section>
```

Comment:
- [ ] Please wrap all the work cards in one section (work section) and use grid to lay out all the cards in the setion.